diff --git a/node_modules/font-tracer/lib/fontTracer.js b/node_modules/font-tracer/lib/fontTracer.js
index 6dff2e2..cbcc744 100644
--- a/node_modules/font-tracer/lib/fontTracer.js
+++ b/node_modules/font-tracer/lib/fontTracer.js
@@ -1011,6 +1011,16 @@ function fontTracer(
     }
   }
 
+  // Purge the custom css properties from the tailwindcss
+  for (const styledText of styledTexts) {
+    styledText.props =
+      Object.keys(styledText.props)
+        .filter(key => !key.startsWith("--tw-")).reduce((acc, key) => {
+          acc[key] = styledText.props[key];
+          return acc;
+        }, {});
+  }
+
   const seenPermutationKeys = new Set();
   const multipliedStyledTexts = _.flatMap(styledTexts, ({ props, ...rest }) =>
     [...expandPermutations(props)]
